### DISCUSSION TITLE
Keyboard shortcut for Expand All and Collapse All

### DISCUSSION THREAD

**Discussion by gmartelly-suki** [nextgenhealthcare/connect#5622] [Feature/Enhancement Ideas]
It would be convenient to have a keyboard shortcut in the Dashboard, Channels, and Message Template views for the Expand All and Collapse All options. Like Ctrl + c for collapse and Ctrl + e for expand.

**Comment by pacmano1** [comment]
Well not ctrl-c which is a break in linux and a copy in other OS but more  keyboard shortcuts would be nice!

**Comment by pladesma** [comment]
This sounds like a nice idea. Can you create a Github issue or convert this discussion into an issue so that we can triage it?

Also, to those that are unfamiliar with the existing Expand All and Collapse All features, you can access them by clicking on the little square button in the top-right corner of the Dashboard, Channels, and Message Template views.
![image](https://user-images.githubusercontent.com/4097281/230199729-7055bfff-64b1-4e80-8d13-b0e682570ed9.png)
